### PR TITLE
Update AssemblyInfo.cs

### PR DESCRIPTION
**Pull request by oolugboy**
# Pull Request Template

Description
Making the internals of the Microsot.Azure.Cosmos project visible to the Microsoft.Azure.Cosmos.CosmosFabric project. This is because we are trying to use/create a custom CosmosQueryResponse class, in a general effort to mock getting a response (with enumerable elements) from requests to the cosmos db store/container. The purpose of mocking requests to the cosmos store, is to further make our Cosmos Fabric testing framework more closely simulate the production environment.

Type of change
Entry into the Assembly.info file.

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/AssemblyInfo.cs [hunk 1]
```diff
@@ -30,4 +30,5 @@
 [assembly: InternalsVisibleTo("CosmosCTL" + AssemblyKeys.ProductPublicKey)]
 [assembly: InternalsVisibleTo("CosmosCTL" + AssemblyKeys.TestPublicKey)]
 [assembly: InternalsVisibleTo("CosmosBenchmark" + AssemblyKeys.ProductPublicKey)]
-[assembly: InternalsVisibleTo("CosmosBenchmark" + AssemblyKeys.TestPublicKey)]
\ No newline at end of file
+[assembly: InternalsVisibleTo("CosmosBenchmark" + AssemblyKeys.TestPublicKey)]
+[assembly: InternalsVisibleTo("Microsoft.Azure.Cosmos.CosmosFabric" + AssemblyKeys.TestPublicKey)]
```
Comment (sourabh1007) [inline Microsoft.Azure.Cosmos/src/AssemblyInfo.cs:34]: @kirankumarkolli @ealsur is it allowed?

**Review by ealsur — CHANGES_REQUESTED** [review]
@oolugboy Can you please share why do you need to mock `QueryFeedResponse`? This is not the public type of any iterator. The iterators return `FeedResponse`.

Please create an Issue to discuss the mocking problem you are facing.

File: Microsoft.Azure.Cosmos/src/AssemblyInfo.cs (hunk 1 above)
Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/AssemblyInfo.cs:34]: For internal types that's the only alternative right now.

**Comment by oolugboy** [comment]
@ealsur , My apologies for my lack of precision in the description. I wanted to create a custom FeedResponse, because I needed to customize how the QueryResponse Resource enumerable was generated from the list of CosmosElements. That is done via this function.
image
The critical steps being the double deserialization that needed to be done in order to properly deserialize/retrieve the expected object.

**Comment by ealsur** [comment]
Closing as not needed

**Pull request closed without merging**